src/components/ItemDetail/DetailSummary/ButtonRow.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const ButtonRow = props => {
+  return (
+    <p>
+      <button id="item-more-details" className="button">
+        More Details
+      </button>
+      <button id="item-cite" className="button">
+        Cite This Item
+      </button>
+    </p>
```

Comment:
Can you wrap this in a `<div>` instead of a `<p>`?  `<p>`'s indicate string text content for screen readers and stuff.